governance-exercises/VoteWizard_executive_exercise_02-08-2023/Executive vote - August 2, 2023.md

```diff
@@ -0,0 +1,157 @@
+
+### Launch Project Funding
+
+As per this forum [post](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/) and the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget), if this executive proposal passes, the following transfer will take place:
+
+- **2 million DAI** will be transferred to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+
+This allocation is consistent with the previously approved [Launch Project budget ](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget).
+
+### Spark Proxy Spell
+
+As per this governance [poll](https://vote.makerdao.com/polling/QmZyFH21) and the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-), if this executive proposal passes, a proxy spell for Spark Protocol at [0x443f3f4328553f5f85dFc0BA3D59969708201E14](https://etherscan.io/address/0x443f3f4328553f5f85dFc0BA3D59969708201E14) will be executed with the following changes:
+
+#### DAI InterestRateStrategy Update
+
+In response to the DSR rate's increase to 8% subsequent to the EDSR's introduction, a new Dai InterestRateStrategy will be activated. This strategy will track the ETH-C stability fee for the borrow rate of Dai in the Spark Protocol, diverging from its previous alignment with the DSR rate. This update is inline with the Stability Scope, which specifies that fees should remain unaffected by the EDSR.
+
+[VoteWizard: The only documentation available on the DAI InterestRateStrategy I found is the smart contract code itself. It has useful comments but are outdated and the voters won't understand it, so I've opted not to include any documentation. Smart Contract code: https://github.com/marsfoundation/sparklend/blob/master/src/DaiJugInterestRateStrategy.sol]
+
+#### Spark Protocol Parameter Changes
+
+- The DAI market [Loan-To-Value (LTV)](https://docs.aave.com/risk/asset-risk/risk-parameters#loan-to-value) will be decreased from 74% to **0.01%**.
+- The DAI market [Liquidation Threshold](https://docs.aave.com/risk/asset-risk/risk-parameters#liquidation-threshold) will be decreased from 76% to **0.01%**.
+
+This will effectively remove DAI as a collateral type, ensuring that there won’t be a situation where the market is at 100% utilization and liquidations on the short positions need to be done. This is particularly important because the sDAI market is frozen, meaning that the only way to short is through the DAI market. Keep in mind that since DAI has active suppliers, LTV can’t be set to zero, this means that usageAsCollateralEnabled will still be true.
```

Comment:
I would probably rephrase this, I know it was pulled from the post / other vote though.